### PR TITLE
Introducing microk8s tutorial

### PR DESCRIPTION
**Pull request by ktsakalozos**
## Done

Adding a tutorial "Introducing microk8s"

Thank you.

### CODE REVIEW

**Comment by pmatulis** [comment]
This looks really good. It will be reviewed shortly.

File: tutorials/containers/introducing-microk8s/introducing-microk8s.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+### Kubernetes dashboard
+
+As we see above the kubernetes-dashboard service in the kube-system namespace has a ClusterIP of `10.152.183.149` and listens on `443`. The ClusterIP is randomly assigned so if you follow these steps on your host, make sure you check the IP you got. Point your browser to `https://10.152.183.149:443` and you will see the kubernetes dashboard UI.
+
+![Kubernetes Dashboard](images/dashboard.png)
+
+### Grafana dashboard
+
+Let's see an alternative approach in accessing hosted services. The API server proxies our services, here is how to get to them:
```
Comment (pmatulis) [inline tutorials/containers/introducing-microk8s/introducing-microk8s.md]: Let's use... to access...?

File: tutorials/containers/introducing-microk8s/introducing-microk8s.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+We only need to point our browser to (http://127.0.0.1:8080/api/v1/namespaces/kube-system/services/monitoring-grafana/proxy)
+
+![Grafana Dashboard](images/grafana.png)
+
+
+## Host your first service in Kubernetes
+Duration: 7:00
+
+As we are introducing microk8s we do not use yaml manifests, instead we create a deployment of microbot with two pods via the cli:
```
Comment (pmatulis) [inline tutorials/containers/introducing-microk8s/introducing-microk8s.md]: You should explain what a manifest is if you bring it up. And why would we not use one here? For simplicity?

File: tutorials/containers/introducing-microk8s/introducing-microk8s.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+kube-system   replicaset.apps/heapster-v1.5.2-5995dc5b8c                  1         1         1       51m
+kube-system   replicaset.apps/heapster-v1.5.2-7864d77c47                  0         0         0       51m
+kube-system   replicaset.apps/heapster-v1.5.2-dd4fd4c49                   0         0         0       51m
+kube-system   replicaset.apps/kube-dns-67b548dcff                         1         1         1       51m
+kube-system   replicaset.apps/kubernetes-dashboard-67d4c89764             1         1         1       51m
+kube-system   replicaset.apps/monitoring-influxdb-grafana-v4-8467db6558   1         1         1       51m
+```
+
+At the very top we have our microbot pods, `service/microbot-service` is the second in the services list. Our service has a ClusterIP through which we can access it. Notice however that our service is of type [NodePort][nodeport]. This means that our deployment is also available on a port on the host machine; that port is randomly selected and in this case it happens to be `32648`. All we need is to point our browser to http://localhost:32648
```
Comment (pmatulis) [inline tutorials/containers/introducing-microk8s/introducing-microk8s.md]: *All we need to do is point...*

Full stop missing at the very end. Use backticks to be consistent with a previous URL.

File: tutorials/containers/introducing-microk8s/introducing-microk8s.md (hunk 2 above)
Comment (ktsakalozos) [reply tutorials/containers/introducing-microk8s/introducing-microk8s.md]: I removed the reference to manifests. There is not point in mention them if we are not going to use them.

**Comment by caldav** [comment]
I'd be keen on a title change, something that would make it a relevant result title when searching for deploying K8s locally. Maybe: "How to deploy a Kubernetes cluster on your workstation", unless cluster is not the right term. Thoughts?

**Comment by pmatulis** [comment]
There is already "Install Kubernetes with conjure-up". How about "Install Kubernetes with microk8s"?

**Comment by ktsakalozos** [comment]
"Install a local Kubernetes with microk8s" ? I would like to differentiate a bit from our CDK offering because microk8s will not give you a production grade cluster.

**Pull request merged**